### PR TITLE
possible `hidden` parameter attribute values

### PR DESCRIPTION
**Pull request by dorschw**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/demisto-sdk/pull/2262

## Description
Describing the possible values for the `hidden` parameter attribute.

### CODE REVIEW

File: docs/integrations/yaml-file.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ An explanation of these fields is as follows:
 | **required** | Boolean value to indicate that the parameter is required |
 | **additionalinfo** | Additional info about the field, will appear under a question mark in the configuration pannel |
 
+Integration parameters may be hidden from the XSOAR UI, using the optional `hidden` field.
+- To hide the parameter in all marketplaces (XSOAR, XSIAM), use a boolean `true`. 
+- To hide the parameter in specific content marketplace versions, provide list of marketplace version names (e.g. `xsoar` or `marketplacev2` for XSIAM)
```
Comment (DeanArbel) [inline docs/integrations/yaml-file.md:92]:
```suggestion
- To hide the parameter in specific content marketplace versions, provide list of marketplace version names (e.g. `[xsoar]` or `[marketplacev2]` for XSIAM)
```

**Review by DeanArbel — APPROVED** [review]

File: docs/integrations/yaml-file.md (hunk 1 above)
Comment (dorschw) [reply docs/integrations/yaml-file.md:92]: the syntax is 
```
hidden:
  -  xsoar
  - marketplacev2
```

which I didn't know how to describe in a one-liner (or whether it's worth a code-block for the example), wdyt?

Comment (DeanArbel) [reply docs/integrations/yaml-file.md:92]: Let's leave it like this.

**Review by DeanArbel — APPROVED** [review]

**Comment by dorschw** [comment]
Will be merged once SDK v1.7.2 (where this feature is implemented) is released

**Comment by MosheEichler** [comment]
The pr is waiting for the release of demisto-sdk 1.7.2

**Pull request merged**